src/jobs/test-nodejs.yml

```diff
@@ -14,8 +14,19 @@ parameters:
     type: steps
     default:
       - run: npm test
+  lint:
+    default: true
```

Comment:
I believe we should consider this a breaking change for this version. This will introduce a default requirement that any consumer of this orb requires `npm run lint` to operate and exist. 

I also believe that this is the right behavior and a welcomed change.

Reply:
Good call.